File: LightWorkApp/src/app/bench.ts
export class Bench {
    id: string;
    title:string;
    kilos:string;
    reps:string;
    username:string;


    constructor(id: string, title:string, kilos:string,reps:string,username:string){
        this.id = id;
        this.title= title;
        this.kilos= kilos;
        this.reps = reps;
        this.username = username;
    }
}

File: LightWork/src/main/java/com/example/Lightwork/bench/Bench.java
package com.example.Lightwork.bench;

import jakarta.validation.constraints.NotEmpty;
import jakarta.validation.constraints.Positive;

public record Bench(Integer id,
                    @NotEmpty
                    String title,
                    @Positive
                    Float kilos,
                    @Positive
                    Integer reps) {
}
